pages/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react'
+import Image from 'next/image';
+import Link from 'next/link';
+
+const Navbar = () => {
+  const GITHUB_LOGO = "/icons/github-white.svg";
+  const REALDEVSQUAD_IMAGE="/images/Real-Dev-Squad@1x.svg";
+  let prev=false;
+  const [toggle, setToggle] = useState<boolean>(false);
+  const sidebarToggle = () => {
+    setToggle(!toggle);
+  };
+
+  return (
+    <div className="font-bold text-base font-sans sticky z-1 insert-0">
+        <nav className="bg-[#1d1283]">
+          <div className="laptop:hidden h-[70px] w-[70px] p-[18px] inline-block cursor-pointer text-[20px] text-white" onClick={sidebarToggle}>
+            <span className="block w-[25px] h-[3px] my-[5px] mx-auto bg-[#fff]"></span>
+            <span className="block w-[25px] h-[3px] my-[5px] mx-auto bg-[#fff]"></span>
+            <span className="block w-[25px] h-[3px] my-[5px] mx-auto bg-[#fff]"></span>
+          </div>
+          <div className="laptop:hidden block m-5 absolute top-0 right-0">
+            <Link
+              className='p-0 m-5.5'
+              href="#"
+            >
+              <button className="rounded-md p-[5px] bg-[#1d1283] text-white cursor-pointer border-2 border-solid border-white flex justify-around" >

```

Comment:
please add an `onClick` here which has an arrow function and when clicked throws error saying function not declared